### PR TITLE
fix GH markdown causing rendering bug

### PR DESCRIPTION
**Pull request by sblack4**
#594

The yaml in bash toward the bottom of this document was being rendered incorrectly on the docs site. I changed the style to be in line with [your websockets page](https://azure.github.io/application-gateway-kubernetes-ingress/how-tos/websockets/) which renders it correctly. 

image

Also the indentation under [Install AAD Pod Identity](https://azure.github.io/application-gateway-kubernetes-ingress/setup/install-new/#install-aad-pod-identity) had the bulleted list being rendered as a code block. 

image

### CODE REVIEW

File: docs/setup/install-new.md [hunk 1]
```diff
@@ -174,8 +174,9 @@ Kubernetes. We will leverage it to install the `application-gateway-kubernetes-i
 Now that we have App Gateway, AKS, and AGIC installed we can install a sample app
 via [Azure Cloud Shell](https://shell.azure.com/):
 
- ```bash
```
Comment (akshaysngupta) [inline docs/setup/install-new.md]: can you revert this ?

Comment (sblack4) [reply docs/setup/install-new.md]: Did it not fix the rendering? If you look at the picture none of that is being rendered as a code block

Comment (sblack4) [reply docs/setup/install-new.md]: But I can revert it no problem if need be

Comment (akshaysngupta) [reply docs/setup/install-new.md]: as in this part, `cat <<EOF | kubectl apply -f -` and `EOF` part.
Formatting change above looks good.

Comment (sblack4) [reply docs/setup/install-new.md]: done 😄

**Pull request merged**